forc-plugins/forc-doc/src/render.rs

```diff
@@ -53,7 +57,7 @@ impl RenderedDocumentation {
             rendered_docs.0.push(RenderedDocument {
                 module_info: doc.module_info.clone(),
                 html_filename: doc.html_filename(),
-                file_contents: HTMLString::from(doc.clone().render()?),
+                file_contents: HTMLString::from(doc.clone().render(type_engine.clone())?),
```

Comment:
I remember us having issues with performance because of cloning the type engine and declaration engine all over forc, would this be an issue here?